applications/GeoMechanicsApplication/custom_constitutive/README.md

```diff
@@ -0,0 +1,28 @@
+# Constitutive laws
+
+
+## Incremental linear elastic interface law
+The constitutive law for an interface element relates tractions $\tau$ to relative displacement $\Delta u$.
+Relative displacement for interface element is the differential motion between the two sides of the interface. As a
+consequence the relative displacement has unit of length [$\mathrm{L}$] and the stiffness has unit of force over cubic length [$\mathrm{F/L^3}$].
+
+### Relative displacement and traction
+In 2D plane strain $y$ is the opening/closing direction of the interface, while differential motion in the tangential direction
+gives shear. Like for a continuum, normal behaviour is placed first in the relative displacement and traction vector. Shear 
+is placed after the normal motion or traction.
+
+$$ \Delta u = \begin{bmatrix} \Delta u_y \\ \Delta u_x \end{bmatrix} $$
+
+$$ \tau = \begin{bmatrix} \tau_{yy} \\ \tau_{xy} \end{bmatrix} $$
```

Comment:
This is more style and I only suggest it because sometimes you just want to quickly check the formula and what the parameters stand for. So I would suggest adding the following: (if correct to my understanding)

> Where:

- $\Delta u_𝑦$: Relative displacement in the 𝑦-direction (normal to the interface).
- $\Delta u_𝑥$: Relative displacement in the 𝑥-direction (tangential to the interface).
- $\tau_{𝑦𝑦}$: Traction in the 𝑦-direction (normal to the interface).
- $\tau_{𝑥𝑦}$: Shear traction in the 𝑥-direction (tangential to the interface).

Reply:
Done.